packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -182,12 +213,29 @@ const start = async (zcf, privateArgs) => {
     zcf.reallocate(offerToSeat, collateralSeat);
   };
 
+  const getShortfallReportInvitation = () => {
+    const shortfallReportOffer = () => {
```

Comment:
non-blocking suggestion for consistency https://github.com/Agoric/agoric-sdk/pull/5179/files#diff-46a4c9227ae419f47fed07515b2459edf285ccf8aee937434e3471b2ea6cabb6

```suggestion
  const maketShortfallReportInvitation = () => {
    const handleShortfallReportOffer = () => {
```

Reply:
I changed to `makeShortfallReportingInvitation` and `handleShortfallReportingOffer`, but that still has `make` and `handle`.

It's not so much an invitation about reports, as it's about reporting.